src/webfaf/blueprints/symbol_transfer.py

```diff
@@ -39,7 +39,7 @@ def process_symbol(build_id, path, offset, problem_type, create_symbol_auth_key)
             h = sha1()
             h.update("symbol_transfer_dummy")
             h.update(problem_type)
-            dummy_report_hash = h.hexdigest()
+            dummy_report_hash = h.encode("utf-8").hexdigest()
```

Comment:
webfaf/blueprints/symbol_transfer.py:42:32: E1101: Instance of 'sha1' has no 'encode' member (no-member)

Reply:
Nice catch - fixed :)